pico_w/wifi_vsys/wifi_vsys.cpp

```diff
@@ -0,0 +1,162 @@
+#include <stdio.h>
+#include "pico/stdlib.h"
+#include "pico/cyw43_arch.h"
+#include "hardware/gpio.h"
+#include "hardware/adc.h"           //to read ADC for soil moisture sensor, connected to pin 36 (3.3V), pin 33 (GND/AGND), pin 34 (GPIO28/ADC2)
```

Comment:
soil moisture sensor - huh?

Reply:
one of these: https://arduino-tutorials.net/tutorial/capacitive-soil-moisture-sensor-arduino
I left the sensor reading in there to make the example at least a little more interesting.